README.md

```diff
@@ -53,27 +56,47 @@ You can install the app using one of the following methods:
 Build the app from sources in a few simple steps:
 
 1. Clone the repository:
-```sh
-git clone https://github.com/fingerprintjs/fingerprint-device-intelligence-android-demo.git
-```
-2. Open up the cloned repository in Android Studio
-3. Open up the `app/src/main/java/com/fingerprintjs/android/fpjs_pro_demo/constants/Credentials.kt` file and assign your Public API Key and Endpoint URL accordingly:
-```kotlin
-import com.fingerprintjs.android.fpjs_pro.Configuration
-
-object Credentials {
-    val apiKey: String = "your_api_key"
-    val endpointUrl: String = Configuration.Region.US.endpointUrl
-}
-```
-You can find the Public API Key in your [dashboard](https://dashboard.fingerprint.com/) at `App Settings` > `API Keys`.
+    ```sh
+    git clone https://github.com/fingerprintjs/fingerprint-device-intelligence-android-demo.git
+    ```
+2. Open the cloned repository in Android Studio.
+3. Open the file `app/src/main/java/com/fingerprintjs/android/fpjs_pro_demo/constants/Credentials.kt` and replace the value for apiKey with your Public API Key. You can find the Public API Key in your [dashboard](https://dashboard.fingerprint.com/) at `App Settings` > `API Keys`.
+   ```kotlin
+    import com.fingerprintjs.android.fpjs_pro.Configuration
+    
+    object Credentials {
+        val apiKey: String = "your_api_key"
+    }
+    ```
+4. Replace the value for endpointUrl with one of our [endpoints](https://dev.fingerprint.com/docs/android-sdk#region-1) or with your custom endpoint, as applicable.
+    ```kotlin
+    import com.fingerprintjs.android.fpjs_pro.Configuration
+    
+    object Credentials {
+        val endpointUrl: String = Configuration.Region.US.endpointUrl
+    }
+    ```
+    
 > [!NOTE]
 > The `debug` and `releaseLocalSign` variants of the app contain a button inside the app bar allowing you to stub the response of the Fingerprint Pro SDK. This might be useful in case you don't want to setup your own API key and region, but still want to check out how the UI of the app looks like on your device.
 4. In "Build Variants" tool window, select either `debug` or `releaseLocalSign` build variant
 > [!NOTE]
-> There is also `release` variant for the internal use. It's very similar to `releaseLocalSign`, but signs the app with another signature and disables the stubbing functionality described above.
+> There is also a `release` variant for testing. It's very similar to the `releaseLocalSign`, but signs the app with another signature and disables the stubbing functionality described above.
```

Comment:
What does "another signature" mean? How is it different from the signature used for `releaseLocalSign`?

Reply:
releaseLocalSign's signature == "dummy" signature. Everyone can sign the app with this signature locally. You can't run the app on the device without signing it first, so this signature is needed if you want to build the "release-like" version of the app (in a sense that it has all the same optimisations and thus runs as fast) from sources and then test it.

release's signature == "production" signature. The app is signed with this signature when being uploaded to the Play Market. It is protected with a password that only our team possesses.

Reply:
@Sergey-Makarov - Thank you for explaining. In essence, if a developer clones this repo, he/she will be able to publish to the PlayStore if they replace "release.jks" with their own keystore file and set their own values for the corresponding environment variables. Correct?